cli/repo/repo_update.go

```diff
@@ -87,8 +92,31 @@ func repoUpdate(ctx context.Context, c *cli.Command) error {
 	if c.IsSet("trusted") {
 		patch.IsTrusted = &trusted
 	}
+	// TODO: remove isGated in next major release
 	if c.IsSet("gated") {
-		patch.IsGated = &gated
+		if gated {
+			patch.RequireApproval = &woodpecker.RequireApprovalAllEvents
+		} else {
+			patch.RequireApproval = &woodpecker.RequireApprovalForks
```

Comment:
Changing the behavior for `gated == false` is also breaking and should be avoided if this PR should go into 2.7.x

Reply:
Lets discuss in the other thread.